### PR TITLE
Fix build warnings in src/xrEngine

### PR DESCRIPTION
**Pull request by Masterkatze**

### CODE REVIEW

File: src/xrEngine/IGame_Persistent.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ bool IGame_Persistent::IsMainMenuActive()
 
 bool IGame_Persistent::MainMenuActiveOrLevelNotExist()
 {
-    return !g_pGameLevel || g_pGamePersistent->m_pMainMenu && g_pGamePersistent->m_pMainMenu->IsActive();
+    return !g_pGameLevel || (g_pGamePersistent->m_pMainMenu && g_pGamePersistent->m_pMainMenu->IsActive());
```
Comment (Xottab-DUTY) [inline src/xrEngine/IGame_Persistent.cpp]: Did we agreed on disabling that warning or leaving it enabled?...

File: src/xrEngine/xr_level_controller.cpp [hunk 2]
```diff
@@ -354,25 +354,25 @@ keyboard_key keyboards[] =
...
     { "kHIRAGANA",              SDL_SCANCODE_LANG4,              "Hiragana" },
     { "kZENHANKAKU",            SDL_SCANCODE_LANG5,              "Zen-Han-kaku" },
-    { "kLANG6",                 SDL_SCANCODE_LANG6               /* Give a better name? */ },
-    { "kLANG7",                 SDL_SCANCODE_LANG7               /* Give a better name? */ },
-    { "kLANG8",                 SDL_SCANCODE_LANG8               /* Give a better name? */ },
-    { "kLANG9",                 SDL_SCANCODE_LANG9               /* Give a better name? */ },
+    { "kLANG6",                 SDL_SCANCODE_LANG6,              "kLANG6" /* Give a better name? */ },
+    { "kLANG7",                 SDL_SCANCODE_LANG7,              "kLANG7" /* Give a better name? */ },
+    { "kLANG8",                 SDL_SCANCODE_LANG8,              "kLANG8" /* Give a better name? */ },
+    { "kLANG9",                 SDL_SCANCODE_LANG9,              "kLANG9" /* Give a better name? */ },
```
Comment (Xottab-DUTY) [inline src/xrEngine/xr_level_controller.cpp:375]: This breaks check for an empty string! Fuck this warning, honestly!

Comment (Xottab-DUTY) [reply src/xrEngine/xr_level_controller.cpp:375]: Applied a bit of emotional spice here, sorry! :D

File: src/xrEngine/IGame_Persistent.cpp (hunk 1 above)
Comment (Masterkatze) [reply src/xrEngine/IGame_Persistent.cpp]: Yes, this one just slipped into this PR :) removed

**Pull request merged**